portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,19 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random activity I might be doing to the page
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addActivity() {
+  const activityContainer = document.getElementById('activity-container');
+
+  // Always show a new activity from the below list of activities
+  // every time the button is clicked.
+  const activities =
```

Comment:
Consider making the list of possible activities easier to find by moving it to the top of the function or even putting it outside the function at the top of the file (after the copyright/license notice). 

At the top of the function you would have:
`const activities = [...]`

And here you would put:
`const eligibleActivities = activities.filter(...)`

(This should also help reduce the line length. There is no strict line length limit in the style guide, but it's good to break up lines at logical boundaries, either by creating a separate statement or adding a line break with indentation.)

Reply:
Good catch - thanks!